db/migrate/20241205013747_add_object_updated_at_to_work.rb

```diff
@@ -0,0 +1,8 @@
+class AddObjectUpdatedAtToWork < ActiveRecord::Migration[8.0]
+  def change
+    add_column :works, :object_updated_at, :datetime
+    add_index :works, :object_updated_at
+
+    Work.all.find_each {|work| work.update!(object_updated_at: work.updated_at) }
```

Comment:
Is this a reversible migration with this line in it? We might want to tweak this migration. (I know we seldom roll back migrations in general, so maybe it's no biggie?) https://guides.rubyonrails.org/active_record_migrations.html#using-the-change-method

Reply:
Rollback works fine:
```
bin/rails db:rollback:primary

== 20241205013747 AddObjectUpdatedAtToWork: reverting =========================
-- remove_index(:works, :object_updated_at)
   -> 0.0112s
-- remove_column(:works, :object_updated_at, :datetime)
   -> 0.0025s
== 20241205013747 AddObjectUpdatedAtToWork: reverted (0.0828s) ================
```

Reply:
@justinlittman If the `:object_updated_at` column is removed, won't the `update!` call raise? Assuming you have works in the DB, of course.

Reply:
I don't have detailed knowledge about how Rails migrations works; however, it worked locally with works in the database.

Reply:
OK, excellent.